src/main/groovy/org/testobject/gradle/TestObjectExtension.groovy

```diff
@@ -2,14 +2,19 @@ package org.testobject.gradle
 
 class TestObjectExtension {
 
-	String baseUrl = "https://app.testobject.com/api/rest"
-	String username
-	String password
-	String team
-	String app
-	Long testSuite
+    String baseUrl = "https://app.testobject.com/api/rest"
+    String username
+    String password
+    String team
+    String app
+    Long testSuite
+    List<String> tests
```

Comment:
i would have expected an dedicated pogo for filters instead of mixing it with the existing parameters, what do you think?